CHANGELOG.md

```diff
@@ -1,8 +1,18 @@
 # Changelog
 
+## 2.1.0
+* BUGFIX: Fixed Raven.js rejecting frames w/ blob URLs. See: https://github.com/getsentry/raven-js/issues/463
+* BUGFIX: Fixed plugin files not consumable without module loader. See: https://github.com/getsentry/raven-js/issues/446
+* BUGFIX: Fixed bug in console.js plugin where `level` wasn't passed. See: https://github.com/getsentry/raven-js/pull/474
+* BUGFIX: Fixed broken debug logging in IE9 and below. See: https://github.com/getsentry/raven-js/pull/473
+* BUGFIX: Fixed `XMLHttpRequest` wrapper not capturing all event handlers. See: https://github.com/getsentry/raven-js/issues/453
+* BUGFIX: `Raven.uninstall` now restores original builtin functions (e.g. setTimeout). See: https://github.com/getsentry/raven-js/issues/228
+* CHANGE: `maxMessageLength` defaults to 0 (no limit). See: https://github.com/getsentry/raven-js/pull/441
+* NEW: `Raven.showReportDialog` (experimental). See: https://github.com/getsentry/raven-js/pull/456
+
```

Comment:
Would be worth noting the `Error.stackTraceLimit` being set.

https://github.com/getsentry/raven-js/pull/419